packages/api/cms-api/src/user-permissions/user-permissions.module.ts

```diff
@@ -6,15 +6,24 @@ import { UserPermission } from "./entities/user-permission.entity";
 import { UserResolver } from "./user.resolver";
 import { UserContentScopesResolver } from "./user-content-scopes.resolver";
 import { UserPermissionResolver } from "./user-permission.resolver";
-import { USER_PERMISSIONS_OPTIONS } from "./user-permissions.constants";
+import { USER_PERMISSIONS_OPTIONS, USER_PERMISSIONS_SERVICE } from "./user-permissions.constants";
 import { UserPermissionsService } from "./user-permissions.service";
 import { UserPermissionsAsyncOptions, UserPermissionsOptions, UserPermissionsOptionsFactory } from "./user-permissions.types";
 
 @Global()
 @Module({
     imports: [MikroOrmModule.forFeature([UserPermission, UserContentScopes])],
-    providers: [UserPermissionsService, UserResolver, UserPermissionResolver, UserContentScopesResolver, UserPermissionsService],
-    exports: [UserPermissionsService],
+    providers: [
+        UserPermissionsService,
+        UserResolver,
+        UserPermissionResolver,
+        UserContentScopesResolver,
+        {
+            provide: USER_PERMISSIONS_SERVICE,
```

Comment:
Why use a constant at all? You could also provide the service using default shorthand syntax for services? https://docs.nestjs.com/fundamentals/custom-providers#standard-providers